google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -1440,4 +1445,49 @@ public void testReadCompressedBlob() throws IOException {
     }
     blob.delete();
   }
+  
+  @Test
+  public void testBucketPolicy() {
+    String projectId = remoteStorageHelper.getOptions().getProjectId();
+    Identity projectOwner = Identity.projectOwner(projectId);
+    Identity projectEditor = Identity.projectEditor(projectId);
+    Identity projectViewer = Identity.projectViewer(projectId);
+    Map<com.google.cloud.Role, Set<Identity>> bindingsWithoutPublicRead =
+    	ImmutableMap.of(
+    	    StorageRoles.legacyBucketOwner(), (Set<Identity>) newHashSet(projectOwner, projectEditor),
+    	    StorageRoles.legacyBucketReader(), newHashSet(projectViewer));
+    Map<com.google.cloud.Role, Set<Identity>> bindingsWithPublicRead =
+        ImmutableMap.of(
+    	    StorageRoles.legacyBucketOwner(), (Set<Identity>) newHashSet(projectOwner, projectEditor),
+    	    StorageRoles.legacyBucketReader(), newHashSet(projectViewer),
+          StorageRoles.legacyObjectReader(), newHashSet(Identity.allUsers()));
+
+    // Validate getting policy.
+    Policy currentPolicy = storage.getPolicy(BUCKET);
```

Comment:
(I should add that my test project is whitelisted and these tests did pass there, so the efficacy of the tests has been verified).